src/main/java/de/bonndan/nivio/output/jgraphx/FinalGraph.java

```diff
@@ -9,31 +9,47 @@
 import com.mxgraph.view.mxGraph;
 import com.mxgraph.view.mxStyleRegistry;
 import com.mxgraph.view.mxStylesheet;
+
 import de.bonndan.nivio.model.*;
 import de.bonndan.nivio.output.*;
+import de.bonndan.nivio.output.Rendered;
 import de.bonndan.nivio.util.RootPath;
+
+import de.bonndan.nivio.output.Icon;
+import de.bonndan.nivio.output.IconService;
+import de.bonndan.nivio.output.LocalServer;
+
```

Comment:
Why do you import `Rendered`, `Icon`, `IconService` and `LocalServer `specifically if you already import `de.bonndan.nivio.output.*`?